docs/how-to/custom-framing.md

```diff
@@ -29,11 +32,26 @@ For more information on working with subtopologies, see the [Subtopologies Guide
 
 To implement a custom framing protocol in F´, will need to implement the following:
 - **Framer**: A component that wraps payload data into frames for transmission.
-- **Deframer**: A component that unpacks received frames, extracts the payload data, and validate the frame.
-- **FrameDetector** (optional): A helper class that detects the start and end of frames in a stream of data, if your transport is stream-based (e.g., TCP, UART).
+- **Deframer**: A component that unpacks received frames, extracts the payload data, and validates the frame.
+- **FrameDetector** Helper Class (optional): A C++ helper class (not an FPP component) that detects the start and end of frames in a stream of data, if your transport is stream-based (e.g., TCP, UART).
```

Comment:
F Prime component.

Reply:
I would say "if your transport does not guarantee complete packets (e.g. TCP UART)